### PR TITLE
Feat/1789 prod gov banner

### LINKED ISSUE #1789: As tech lead, I need the govbanner changed for production environment
**Description:**
_Currently, TDP's prod frontend says the following:

https://github.com/HHS/TANF-app/blob/a913fb896e2558fa2625ce655f2696f8e5f1b973/tdrs-frontend/src/components/GovBanner/GovBanner.jsx#L24

govbanner

For routes in lower environments, this is fine, but for production, it should look like this [ref](https://designsystem.digital.gov/components/banner/): 

```
An official website of the United States government
```

**Acceptance Criteria:** 
- [ ] govbanner should say "An official website of the United States government" in prod
- [ ] govbanner should stay unchanged in lower environments 
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
- [ ] update [`GovBanner.jsx`](https://github.com/raft-tech/TANF-app/blob/develop/tdrs-frontend/src/components/GovBanner/GovBanner.jsx)
- [ ] Run Testing Checklist and confirm all tests pass

**Supporting Documentation:**
- https://designsystem.digital.gov/components/banner/

### PR DESCRIPTION
**Pull request by riatzukiza**
## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #1789 

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. commit this patch: https://github.com/raft-tech/TANF-app/pull/1907/files#r918196208
2. Deploy to a development space with a free label
3. Check that the banner says "An Official Website of the United States government."
4. Revert the commit when you are done

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
- [x] govbanner should say "An official website of the United States government" in prod
- [x] govbanner should stay unchanged in lower environments 
+ [x] **`lfrohlich`** and/or **`adpennington`**  confirmed that ACs are met.

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
  + [ ] If this PR introduces backend code changes, are they meaningfully tested?
  + [x] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [x] Frontend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)
  + [ ] Backend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)

+ [x] Are backend code style checks passing on CircleCI?
+ [x] Are frontend code style checks passing on CircleCI?
+ [x] Are code maintainability principles being followed?

### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [ ] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?

### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)

+ [x] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)

+ [ ] Does this PR provide background for why coding decisions were made?
+ [ ] If this PR introduces backend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [x] If this PR introduces frontend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?
+ [x] Can reviewer explain and take ownership of these elements presented in this code review?

### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)

+ [x] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any new security issues?
+ [ ] If new issues detected, is investigation and/or remediation plan documented? 

### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
+ [ ] the purpose of the research
+ [ ] methods used to conduct the research 
+ [ ] who participated in the research
+ [ ] what was tested and how
+ [ ] impact of research on TDP
+ [ ] (_if applicable_) final design mockups produced for TDP development

### CODE REVIEW

File: tdrs-frontend/src/components/GovBanner/GovBanner.test.js [hunk 1]
```diff
@@ -9,4 +9,82 @@ describe('GovBanner', () => {
...
+  describe('Environmental behavior', () => {
+    const OLD_ENV = process.env
+    beforeEach(() => {
+      jest.resetModules()
+      process.env = { ...OLD_ENV }
+    })
+    afterAll(() => {
+      process.env = OLD_ENV
+    })
+    test('displays correct behavior in development', () => {
```
Comment (andrew-jameson) [inline tdrs-frontend/src/components/GovBanner/GovBanner.test.js:21]:
```suggestion
    test('Banner adapts behavior to being in local developer environments', () => {
```

File: tdrs-frontend/src/components/GovBanner/GovBanner.test.js [hunk 2]
```diff
@@ -9,4 +9,82 @@ describe('GovBanner', () => {
...
+
+      const wrapper = shallow(<GovBanner />)
+      expect(
+        wrapper
+          .text()
+          .includes('A Test Demo website of the United States government')
+      ).toBe(true)
+    })
+
+    test('displays correct behavior in dev', () => {
```
Comment (andrew-jameson) [inline tdrs-frontend/src/components/GovBanner/GovBanner.test.js:45]:
```suggestion
    test('Banner adapts behavior to being in cloud dev environments', () => {
```

File: tdrs-frontend/src/components/GovBanner/GovBanner.test.js [hunk 3]
```diff
@@ -9,4 +9,82 @@ describe('GovBanner', () => {
...
+
+      const wrapper = shallow(<GovBanner />)
+      expect(
+        wrapper
+          .text()
+          .includes('A Dev Demo website of the United States government')
+      ).toBe(true)
+    })
+
+    test('displays correct behavior in staging', () => {
```
Comment (andrew-jameson) [inline tdrs-frontend/src/components/GovBanner/GovBanner.test.js:57]:
```suggestion
    test('Banner adapts behavior to being in cloud staging environments', () => {
```

File: tdrs-frontend/src/components/GovBanner/GovBanner.test.js [hunk 4]
```diff
@@ -9,4 +9,82 @@ describe('GovBanner', () => {
...
+      process.env.REACT_APP_CF_SPACE = 'tanf-prod'
+      const wrapper = shallow(<GovBanner />)
+      expect(
+        wrapper
+          .text()
+          .includes('An Official website of the United States government')
+      ).toBe(true)
+    })
+
+    test('displays correct behavior in some other hypothetical yet to exist environment', () => {
```
Comment (andrew-jameson) [inline tdrs-frontend/src/components/GovBanner/GovBanner.test.js:79]:
```suggestion
    test('Banner adapts behavior to being in some other hypothetical yet to exist environment', () => {
```

**Review by reitermb — APPROVED** [review]
Signing off on a11y, everything's still working as it should on the banner!

File: tdrs-frontend/src/components/GovBanner/GovBanner.test.js [hunk 5]
```diff
@@ -9,4 +9,82 @@ describe('GovBanner', () => {
...
+
+      const wrapper = shallow(<GovBanner />)
+      expect(
+        wrapper
+          .text()
+          .includes('A Test Demo website of the United States government')
+      ).toBe(true)
+    })
+
+    it('displays correct behavior in dev', () => {
```
Comment (riatzukiza) [inline tdrs-frontend/src/components/GovBanner/GovBanner.test.js:45]: be more specific

File: scripts/deploy-frontend.sh [hunk 6]
```diff
@@ -18,6 +19,7 @@ update_frontend()
 
     echo "REACT_APP_BACKEND_URL=https://$CGHOSTNAME_BACKEND.app.cloud.gov/v1" >> .env.production
     echo "REACT_APP_BACKEND_HOST=https://$CGHOSTNAME_BACKEND.app.cloud.gov" >> .env.production
+    echo "REACT_APP_CF_SPACE=$CF_SPACE" >> .env.production
```
Comment (riatzukiza) [inline scripts/deploy-frontend.sh:22]:
```suggestion
    echo "REACT_APP_CF_SPACE=tanf-prod" >> .env.production
```

**Review by n0remac — APPROVED** [review]

File: scripts/deploy-frontend.sh [hunk 7]
```diff
@@ -19,7 +19,7 @@ update_frontend()
 
     echo "REACT_APP_BACKEND_URL=https://$CGHOSTNAME_BACKEND.app.cloud.gov/v1" >> .env.production
     echo "REACT_APP_BACKEND_HOST=https://$CGHOSTNAME_BACKEND.app.cloud.gov" >> .env.production
-    echo "REACT_APP_CF_SPACE=$CF_SPACE" >> .env.production
+    echo "REACT_APP_CF_SPACE=tanf-prod" >> .env.production
```
Comment (ADPennington) [inline scripts/deploy-frontend.sh:22]: @riatzukiza you can revert this change.
